### DISCUSSION TITLE
VSCode extension

### DISCUSSION THREAD

**Discussion by tristan-f-r** [pest-parser/pest#771] [Ideas]
There should be an officially maintained vscode extension for pest to enable syntax highlighting (and possibly a playground) for making pest grammers.

**Comment by tomtau** [comment]
@Jamalam360 mentioned on Discord he'd be interested in working on this, but not sure if he'd started anything yet.

**Reply by Jamalam360** [reply]
Mhm, still interested potentially, but I don't think we ever figured out the best way to go about it. I suggested an LSP that both extensions could use but that may be overkill

**Comment by tomtau** [comment]
https://github.com/pest-parser/pest-ide-tools
